CONTRIBUTING.md

```diff
@@ -7,7 +7,10 @@
 
 # Prerequisites:
 
-Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+- Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
```

Comment:
Why not Community?

Reply:
Also right here, it should be Community or higher.